frontend/src/services/transactions/utils.ts

```diff
@@ -1,10 +1,46 @@
-import type { Block, JsonRpcSigner } from '@ethersproject/providers';
+import type { Block, JsonRpcSigner, Listener, Provider } from '@ethersproject/providers';
 import { JsonRpcProvider } from '@ethersproject/providers';
-import type { ContractInterface } from 'ethers';
+import type { ContractInterface, Event } from 'ethers';
 import { Contract } from 'ethers';
 
 import type { EthereumAddress } from '@/types/data';
 
+type ConfirmationTimeMap = {
+  [chainId: string]: number;
+};
+
+const DEFAULT_CONFIRIMATION_TIME_BLOCKS = 1;
```

Comment:
From the official ethersjs documentation on what number of confirmations blocks means:
>The number of blocks that have been mined (including the initial block) since this transaction was mined.

So, 1 would mean waiting until the block where the transaction is included in has been mined. 

I would keep it to 1 here so we make sure we wait until the transaction has been included in a mined block.

Reply:
Ok, then it's perfect how it is!